skiko/src/iosMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.ios.kt

```diff
@@ -33,14 +34,33 @@ internal class MetalRedrawer(
     private val queue = device.newCommandQueue()!!
     private var currentDrawable: CAMetalDrawableProtocol? = null
     private val metalLayer = MetalLayer()
+    private var activeFrameSubscription: AtomicBoolean = atomic(false)
```

Comment:
I'm wondering if we need atomic.
we have `isDisposed` and it's just a boolean var.
But I'm not sure if we have some guarantees about the calling thread(s). 

Maybe we can add the display link in `init` block only once and then manage it by setting true/false to `isPaused` https://developer.apple.com/documentation/quartzcore/cadisplaylink/1621229-ispaused - it's thread-safe too (maybe it's cheaper than AtomicBoolean from kotlin)

Then  in `dispose` we will need to call `invalidate()` https://developer.apple.com/documentation/quartzcore/cadisplaylink/1621293-invalidate

Reply:
If we can implement `isPaused` approach, it looks better.

Reply:
Good suggestion!
Done.